services/galaxytoolshed/galaxytoolshed-activity.service.js

```diff
@@ -42,6 +37,6 @@ export default class GalaxyToolshedCreatedDate extends BaseGalaxyToolshedService
       owner,
     })
     const { create_time: date } = response[0]
-    return this.constructor.render({ date })
+    return renderDateBadge(date)
```

Comment:
I think we should reverse color order here.
Notice how originaly we used blue, as this is not last update but *creation* date.
I think older could be better here as it is indicates stability. Otherwise we might want to stay with blue.

Reply:
Agreed. https://github.com/badges/shields/pull/10682/commits/9ce8948f029bac2fd0335059890889931a720aaa